ibm/data_source_ibm_is_vpn_gateway_connection.go

```diff
@@ -0,0 +1,548 @@
+										Description: "Link to documentation about deleted resources.",
+									},
+								},
+							},
+						},
+						"href": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The IPsec policy's canonical URL.",
+						},
+						"id": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The unique identifier for this IPsec policy.",
+						},
+						"name": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The user-defined name for this IPsec policy.",
+						},
+						"resource_type": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The resource type.",
+						},
+					},
+				},
+			},
+			"mode": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The mode of the VPN gateway.",
+			},
+			"name": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The user-defined name for this VPN gateway connection.",
+			},
+			"peer_address": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The IP address of the peer VPN gateway.",
+			},
+			"psk": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The preshared key.",
+			},
+			"resource_type": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The resource type.",
+			},
+			"status": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "The status of a VPN gateway connection.",
+			},
+			"routing_protocol": {
+				Type:        schema.TypeString,
+				Computed:    true,
+				Description: "Routing protocols are disabled for this VPN gateway connection.",
+			},
+			"tunnels": {
+				Type:        schema.TypeList,
+				Computed:    true,
+				Description: "The VPN tunnel configuration for this VPN gateway connection (in static route mode).",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"public_ip": {
+							Type:        schema.TypeList,
+							Computed:    true,
+							Description: "The IP address of the VPN gateway member in which the tunnel resides.",
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"address": {
+										Type:        schema.TypeString,
+										Computed:    true,
+										Description: "The IP address.This property may add support for IPv6 addresses in the future. When processing a value in this property, verify that the address is in an expected format. If it is not, log an error. Optionally halt processing and surface the error, or bypass the resource on which the unexpected IP address format was encountered.",
+									},
+								},
+							},
+						},
+						"status": {
+							Type:        schema.TypeString,
+							Computed:    true,
+							Description: "The status of the VPN Tunnel.",
+						},
+					},
+				},
+			},
+			"local_cidrs": {
```

Comment:
How will we support this case VPNGatewayConnectionPolicyMode:
and fill the `local_cidrs` and `peer_cidrs`